web/templates/user/form.html.haml

```diff
@@ -15,4 +15,4 @@
     = password_input f, :password_confirmation, placeholder: "confirm password", class: "br2 ba b--silver pa3 tc w-100 mv1"
     .red.tc= error_tag f, :password_confirmation
   .form-group
-    = submit "Create account", class: "pa3 f5 br2 bg-white ba b--black center db w-third mt4"
+    = submit "Create account", class: "pa3 f5 br2 bg-blue ba b--black center db w-third mt4 light-gray b--transparent"
```

Comment:
When editing a user we should contextually show edit. When creating it should show create.